### ISSUE TITLE
Box-shadow: none doesn't work.

### ISSUE THREAD

**Issue by johnridesabike** [SentiaAnalytics/bs-css#148]
Right now, there doesn't seem to be any elegant way to declare `box-shadow: none`. You have to either use `unsafe("boxShadow", "none")` or ``boxShadows([`shadow("none")])``. The latter feels hacky and isn't really any safer than the "unsafe" method.

I'm not sure how this would easily coexist with the existing `boxShadow` function, though. Maybe there could be a special-case `boxShadowNone` function?

I'm also not sure how many other properties are affected by this same issue. Some similar ones have more elegant workarounds. Example: ``outline(`none)`` is invalid, but ``outlineStyle(`none)`` accomplishes the same result and is fine. Box-shadow, unfortunately, can't be broken into smaller properties (AFAIK).

Edit: After posting this, it occurred to me that perhaps a straightforward way of enabling this functionality is to allow `boxShadows` to accept `` `none`` in addition to `` `shadow``.

**Comment by giraud** [comment]
shadow types need some refactoring. They prevent to abstract the type 'rule'.

**Comment by giraud** [comment]
Resolved with PR #153 

I'm using a phantom type to dissociate a box or a text shadow definition.
Now, box and text shadows are rules and that allows to create these css rules:

```
open Css;

boxShadow(Shadow.box(~x=px(1), ~y=px(2), red));
boxShadows([Shadow.box(yellow), Shadow.box(red)]);
boxShadow(none);
boxShadow(inherit_);
boxShadow(initial);
boxShadow(unset);
important(boxShadow(none));

textShadow(Shadow.text(~blur=vh(1.), red));
textShadows([Shadow.text(yellow), Shadow.text(red)]);
etc...
```

This is a breaking change of course (I don't see how to do differently) and the syntax is a little bit more verbose for single statement:
```
boxShadow(~x=px(1), ~y=px(2), red)
vs
boxShadow(Shadow.box(~x=px(1), ~y=px(2), red));
```

It also resolve issue #147

**Comment by giraud** [comment]
Fixed in 10.0.0
